debian/copyright

```diff
@@ -4,60 +4,228 @@ Upstream-Contact: qgis-developer@lists.osgeo.org
+                       2012, Anita Graser <anitagraser@gmx.at>
+                       2012, Australia Indonesia Facility for Disaster Reduction
+                       2012, Carterix Geomatics
+                       2012, Etienne Tourigny <etourigny.dev@gmail.com>
+                 2008, 2012, Horst Düster
+       2003-2005, 2007-2013, Tim Sutton <tim@linfiniti.com>
+                  2008-2013, Borys Jurgiel <borysiasty@aster.pl>
+                  2012-2013, Chris Crook <ccrook@linz.govt.nz>
+                  2012-2013, Larry Shaffer <larrys@dakcarto.com>
+                  2012-2013, Massimo Endrighi <massimo.endrighi@geopartner.it>
+                  2012-2013, Salvatore Larosa <lrssvtml@gmail.com>
+                  2012-2013, Vinayan Parameswaran <vinayan123@gmail.com>
+                       2013, Alvaro Huarte <ahuarte47@yahoo.es>
+                       2013, CS Systemes d'information (CS SI) <otb@c-s.fr>
+                       2013, Joshua Arnott <josh@snorfalorpagus.net
+                  2012-2013, René-Luc D'Hont <rldhont@3liz.com>
+                  2004-2014, Marco Hugentobler <mhugent@users.sourceforge.net>
+           2005, 2012, 2014, Hugo Mercier <hugo.mercier@oslandia.com>
+            2011-2012, 2014, Nathan Woodrow <madmanwoo@gmail.com>
+            2008-2012, 2014, Jürgen E. Fischer <jef@norbit.de>
+                 2011, 2014, Tamas Szekeres <szekerest@gmail.com>
+                  2009-2014, Alexander Bruy <alexander.bruy@gmail.com>
+                  2012-2014, Matthias Kuhn <matthias.kuhn@gmx.ch>
+                  2012-2014, Piotr Pociask <opengis84@gmail.com>
+            2011, 2013-2014, Bernhard Ströbl <bernhard.stroebl@jena.de>
+                  2013-2014, Martin Isenburg <martin@rapidlasso.com>
+                       2014, Agresta S. Coop <iescamochero@agresta.org>
+                       2014, Alessandro Pasotti <a.pasotti@itopen.it>
+                       2014, Angelos Tzotsos <tzotsos@gmail.com>
+                       2014, Giovanni Allegri
+                       2014, Michael Douchin
+                       2014, Niccolo' Marchi <sciurusurbanus@hotmail.it>
+                       2014, Radoslaw Guzinski <rmgu@dhi-gras.com>
+                       2014, Tom Kralidis <tomkralidis@gmail.com>
+       2004-2006, 2009-2015, Radim Blazek <blazek@itc.it>
+                  2005-2015, Martin Dobias <wonder.sk@gmail.com>
+                  2012-2015, Victor Olaya <volayaf@gmail.com>
+                  2012-2015, The QGIS Project
+                  2012-2015, Denis Rouzaud <denis.rouzaud@gmail.com>
+                  2013-2015, Nyall Dawson <nyall.dawson@gmail.com>
+                  2014-2015, Arnaud Morvan <arnaud.morvan@camptocamp.com>
+                  2014-2015, Sandro Mani <manisandro@gmail.com>
+                  2014-2015, Sandro Santilli <strk@keybit.net>
+                       2015, Michael Kirk <michael.john.kirk@gmail.com>
 License: GPL-2+
```

Comment:
> Are the year necessary?  They are incomplete anyway and for all continouos
> contributors that will be outdated quickly (eg. Tom Kralidis contributed
> also this year)

Yes, copyright-format 1.0 specifies that the Copyright field "collects all
relevant copyright notices for the files of this paragraph. Not all
copyright notices may apply to every individual file, and years of
publication for one copyright holder may be gathered together."

It further specifies that the "Copyright field may contain the original
copyright statement copied exactly (including the word "Copyright"), or it
may shorten the text or merge it with other copyright statements as
described above, as long as it does not sacrifice information."

Removing the copyright years would sacrifice information.

See:
https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#copyright-field

The copyright years were gathered from the copyright headers in the source
files, and grouped by copyright holder. The update copyright file reflects
the copyright statements from the source files quite complete now.

If the changes by Tom Kralidis in 2015 were significant enough the
copyright years in the relevant sources file should be updated, if the
changes were not significant enough to claim copyright the years don't
need an update.